app/src/org/commcare/views/QuestionsView.java

```diff
@@ -340,7 +374,6 @@ public void setBinaryData(Object answer, PendingCalloutInterface pendingCalloutI
         for (QuestionWidget q : widgets) {
             if (questionFormIndex.equals(q.getFormId())) {
                 q.setBinaryData(answer);
-                pendingCalloutInterface.setPendingCalloutFormIndex(null);
```

Comment:
I removed this call because it causes `getPendingWidget` in `FormEntryActivity` to return null at the point that I call it in `restoreFocusToCalloutQuestion`. I'm pretty sure this is fine because `onResumeSessionSafe` of `FormEntryActivity` will always calls `mFormController.setPendingCalloutFormIndex(null)` after returning from the callout. But just wanted to flag in case anyone sees an issue. I think this was just here as a more proactive, but not strictly necessary, place to set the index back to null.